src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs

```diff
@@ -165,4 +165,8 @@
         [incoming-contact-request-view props]]
 
        :else
-       [incoming-contact-request-view props])]))
+       [gesture/touchable-without-feedback
+        {:on-press (fn []
+                     (rf/dispatch [:hide-popover])
```

Comment:
nit: Why are we using :hide-popover in this namespace, Shouldn't the convention be `:navigate-back`?
CC: @J-Son89 @Parveshdhull

Reply:
@briansztamfater @flexsurfer might know

Reply:
I think `:navigate-back` should handle the popover if it's open
maybe it should dispatch `:hide-popover` if there's an open popover

Reply:
I think activity center is a modal, so `:navigate-back` should work for closing it. (or we are using :hide-popover for some other purpose?)

Reply:
nobody knows :)